ext/popup.js

```diff
@@ -71,24 +99,70 @@ Workspace.onChange = function(callback) {
 
 function HtmlBuilder(container) {
   this.container = container;
+  this.header = document.createElement("header");
   this.node = document.createElement("ul");
 }
 
+HtmlBuilder.prototype.generateFavicon = function(favIconUrl, className) {
+  var icon = document.createElement("img");
+
+  icon.classList.add("favicon");
+  if (className) {
+    icon.classList.add(className);
+  }
+
+  icon.src = favIconUrl;
+
+  icon.onerror = () => {
+    icon.src = 'icon.png';
+  };
+
+  return icon;
+};
+
+HtmlBuilder.prototype.buildHeader = function() {
+  var removeLnk = document.createElement("a");
+
+  removeLnk.appendChild(document.createTextNode("Remove all workspaces"));
+  removeLnk.addEventListener("click", function() {
+    chrome.storage.local.clear();
```

Comment:
Why don't use you `Workspace.destroyAll` function instead of inlining the function here?